src/widgets/_toolbars.scss

```
@@ -7,6 +7,22 @@
             rgba(black, 0.07) 1px,
             transparent rem(3px)
         );
+
+    .sidebar & {
+        background-color: mix(bg-color(3), bg-color(4), 50%);
+        background-image:
+            linear-gradient(
+                to bottom,
+                rgba(black, 0.3),
+                rgba(black, 0.04) 1px,
+                transparent rem(3px)
+            );
+        padding: rem(6px);
```

I think this is what's really chonking it up for me. How about:

```suggestion
        padding: rem(3px);
```

---

Can we keep the horizontal padding at 6px so that the icons line up with the ones in the sidebar items?

---

@danrabbit that works in Files but it doesn't line up with anything in Code. Not sure how much that matters. But yeah `3px 6px` is still better than 6px all around.